playbooks/roles/rsyslog/tasks/main.yml

```diff
@@ -1,13 +1,20 @@
 ---
-- name: Copy the modified rsyslog configuration into place that prevents Libreswan and OpenConnect traffic from being logged
+- name: Copy the modified rsyslog configuration into place that prevents Libreswan traffic from being logged
   copy:
-    src: "{{ item }}"
-    dest: /etc/rsyslog.d/{{ item }}
+    src: 40-ipsec.conf
+    dest: /etc/rsyslog.d/40-ipsec.conf
     owner: root
     group: root
     mode: 0644
-  with_items:
-    - 40-ipsec.conf
-    - 41-openconnect.conf
+  notify: Restart rsyslog
+
+- name: Copy the modified rsyslog configuration into place that prevents OpenConnect traffic from being logged
```

Comment:
This is another case where the `.d` directory makes our life easier and the `rsyslog` role doesn't need to know about any of the roles that tweak its configuration. I think it would be better to put the `rsyslog` role into the OpenConnect role's meta dependencies and then have a task in `playbooks/roles/openconnect/tasks/install.yml` that copied the openconnect rsyslog config template into the `/etc/rsyslog.d` directory.